masonite/drivers/CacheDiskDriver.py

```diff
@@ -0,0 +1,30 @@
+import os
+import time
+
+
+class CacheDiskDriver():
+    """
+    Cache from the disk driver
+    """
+
+    def __init__(self, CacheConfig, Application):
+        self.config = CacheConfig
+        self.appconfig = Application
+
+    def store(self, template, location=None):
+        if not location:
+            location = self.config.DRIVERS['disk']['location']
+
+        location += '/'
+
+        # Generate file cache
+        path_filename = os.path.basename(template)
```

Comment:
This should be `path_filename = os.path.join('resources/templates', os.path.basename(template))` because all templates are in the `resources/templates` directory. This should make the tests pass. Only problem is that the cached template is:

`{{ test }}`

which you'll see when you run the tests. It should be the rendered template meaning it should not have any Jinja2 templating in it.